app/views/webhook_deliveries/index.html.haml

```diff
@@ -25,6 +26,9 @@
           %td.error_class= webhook_delivery.error_class
           %td.created_at= webhook_delivery.created_at && DateTime.parse(webhook_delivery.created_at).strftime("%B %d, %Y")
           %td.completed_at= webhook_delivery.completed_at && DateTime.parse(webhook_delivery.completed_at).strftime("%B %d, %Y")
+          %td.actions
+            = link_to 'Redeliver', redeliver_client_application_webhook_delivery_path(params[:client_application_id], webhook_delivery.id), method: :post, data: { confirm: "Are you sure?" }, class: 'btn btn-primary'
+
```

Comment:
Layout on the list page

<img width="800" alt="Screenshot 2024-10-30 at 5 15 55 PM" src="https://github.com/user-attachments/assets/3bc93bd3-4a28-44ed-b652-a307e4e7f7ba">